src/Foundation/NSUrl.cs

```diff
@@ -142,6 +142,19 @@ public bool SetResource (NSString nsUrlResourceKey, NSObject value)
 			return SetResourceValue (value, nsUrlResourceKey, out error);
 		}
 
+		public int Port
```

Comment:
style: opening brace goes on the same line as the property (and for the same line as the `get` keyword as well)